### ISSUE TITLE
Inspiration

### ISSUE THREAD

**Issue by yuvadm** [daat-hamakom/data#6]
http://oridomi.com/

**Comment by mushon** [comment]
http://histography.io/

**Comment by mushon** [comment]
http://io.morphocode.com/urban-layers/

**Comment by mushon** [comment]
[Simple Zoomable Timeline in D3.js](http://jsfiddle.net/uWdkw/)

**Comment by mushon** [comment]
[Timeline.js](http://timeline.knightlab.com/) - Big, bloated and ugly, but still a reference.

**Comment by mushon** [comment]
[Christchurch 2010 Timeline](http://bl.ocks.org/tnightingale/4718717) - D3js Map + Timeline viz

**Comment by josefspr** [comment]
Thanks, the historiography is beautiful. Christchurch is quite elegant and nice to work, seems in our
direction, no ?

**Comment by mushon** [comment]
Here's another timeline inspiration that I forgot about, it functions quite similarly to what we're shooting for (beautiful project):
http://thewhalehunt.org/whalehunt.html

**Comment by josefspr** [comment]
Thanks, the project is charming although I needed to read the interface explanation to fully understand.
It has a nice feel and I like very much the magnification on hovering the timeline.

**Comment by mushon** [comment]
I agree. I think that magnification would be the main takeaway from this reference.
